doc/cla/corporate/tvtmarine-automation.md

```diff
@@ -13,7 +13,7 @@ David Tran david.tran@tvtmarine.com https://github.com/tvtma/
 List of contributors:
 
 David Tran david.tran@tvtmarine.com https://github.com/davidtranhp
-Long Do long.do@tvtmarine.com https://github.com/hoanglong87
```

Comment:
as this email has already made changes (869b833618f727710f75ee61eca508bd3128e181), could you keep the line? If this person no longer works in the company, you can move it at the bottom and add a "(up to 2019-..." message

Reply:
Done!